exercises/chapter8/src/Data/AddressBook.purs

```diff
@@ -17,8 +17,6 @@ data PhoneType
   | CellPhone
   | OtherPhone
 
-derive instance eqPhoneType :: Eq PhoneType
```

Comment:
There's not a good way to to include tests for the UI changes in CI, so no need to edit this file.